App.js

```diff
@@ -14,13 +14,12 @@ ProfileRoute.queries = {
 };
 
 view App {
+  prop onLogout
+
   <Relay.RootContainer
-    {...{ route: new ProfileRoute() }}
+    route={new ProfileRoute()}
```

Comment:
This won't work, this hack was made intentionally so I can pass route without conflicting with Flint route.